app/templates/gentelella/users/events/_table.html

```diff
@@ -42,7 +46,11 @@
                 {% endfor %}
 
                 <tr>
+
+                    <td><a href="{{ url_for('events.details_view', event_id=event.id) }}">  <img src="{{ event.logo }}"  onerror="imgError(this); " style=" width:70px;"><p style ="padding:8px;">{{ event.name }}</p></a></td>
```

Comment:
The line is duplicated here. And only thing that is supposed to have changed in this PR is this line ... How come there are so many changes here ? Please check